source/style/_theme/colors.less

```diff
@@ -168,7 +166,8 @@
       padding: 0;
       margin-top: 0;
       .menu-content {
-        background-color: transparent;
+        //background-color: transparent;
```

Comment:
Strong preference to remove commented code unless there's a compelling reason to keep it around.